spec/controllers/organizations_controller_spec.rb

```diff
@@ -12,4 +32,27 @@
 
     it { expect(response).to render_template('index') }
   end
+
+  describe 'GET #conferences' do
+    before :each do
+      get :conferences, organization_id: organization.id
+    end
+
+    it 'loads the organization' do
+      expect(assigns(:organization)).to eq organization
+    end
+
+    it 'includes organizational conferences' do
```

Comment:
Perhaps easier to understand if it is simply put as `includes organization conferences`